### PR TITLE
Peralta exercise

### PR DESCRIPTION
**Pull request by luis-ahw2019**
This pull request contains the changes related with the problem 1 of the tutorial "Pull Requests and Reviewing Code".

### CODE REVIEW

**Comment by atakan** [comment]
I'll take a look.

File: Peralta/template/simple_functions.py [hunk 1]
```diff
@@ -10,3 +10,19 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
```
Comment (atakan) [inline Peralta/template/simple_functions.py:13]: Well written and clear.

File: Peralta/template/simple_functions.py [hunk 2]
```diff
@@ -10,3 +10,19 @@ def factorial(value):
...
+def is_prime(value):
+    assert type(value) is int
+    
+    # Initialise the result as True
+    result = True
+
+    # Try to find a integer which divides the input value:
+    # If we find one, the result will be changed to False
+    for i in range(2, value):
```
Comment (atakan) [inline Peralta/template/simple_functions.py]: This could be improved by going up to about sqrt(n).

Comment (atakan) [inline Peralta/template/simple_functions.py]: I think n<=1 needs to be checked. It seems like, this function will return True if the input is, e.g., 1.

**Review by atakan — CHANGES_REQUESTED** [review]
There might be a bug. Please check.

File: Peralta/template/simple_functions.py (hunk 1 above)
Comment (luis-ahw2019) [reply Peralta/template/simple_functions.py:13]: Thanks!

File: Peralta/template/simple_functions.py (hunk 2 above)
Comment (luis-ahw2019) [reply Peralta/template/simple_functions.py]: I will implement this improvement :-)

Comment (luis-ahw2019) [reply Peralta/template/simple_functions.py]: You are completely right: now I will check that cases too.

**Comment by luis-ahw2019** [comment]
I think that I have improved the is_prime function as suggested :smile: Thanks for your reviewing!

**Comment by astrofrog** [comment]
Thanks for opening the pull request and for the review(s)!  We will merge this shortly.  Below I have provided a few comments about the *process* of the pull request and review. But note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code.

* @luis-ahw2019 - one minor comment is that the title could have been more descriptive.

* @atakan - you did not "approve" or "request changes" when you submitted the review.  Without that a maintainer might not be sure if your review should be taken as a complete review or just general comments. In this case, 'Request changes' would have been more appropriate than 'Comment'.

**Pull request merged**